### PR TITLE
cleanup code

### PR DESCRIPTION
**Pull request by butlerx**
changhes from #1179 not covered in #1186

### CODE REVIEW

**Comment by Wardormeur** [comment]
Rebase please..
And if we could avoid 80 files commit, that'd be nice : in this scenario, i'd need the devDocker upgrade (because i broke it) without the eslint stuff.

**Comment by butlerx** [comment]
rebased 
also it was only 40 file 😛

File: web/lib/http-error-handler.js [hunk 1]
```diff
@@ -1,33 +1,43 @@
 const _ = require('lodash');
 const Boom = require('boom');
 
-module.exports = function (server) {
-  return function (request, reply) {
-    const response = request.response;
-    let status = _.get(response, 'output.payload.statusCode', undefined);
-    let msg = '';
-    if (!status) status = _.get(response, 'statusCode', 500);
+module.exports = server => (request, reply) => {
```
Comment (Wardormeur) [inline web/lib/http-error-handler.js]: dble arrow ugliness

File: web/lib/http-error-handler.js [hunk 2]
```diff
@@ -1,33 +1,43 @@
...
-        case 403:
-        case 410:
-          return reply.continue();
-          break; // eslint-disable-line no-unreachable
-        default:
-          return reply(Boom.badImplementation());
-      }
-    } else {
+  if (status >= 400) {
+    request.log(
```
Comment (Wardormeur) [inline web/lib/http-error-handler.js]: rewrap, it's not a logic matter, it's a log

File: web/lib/locale.js [hunk 3]
```diff
@@ -1,25 +1,22 @@
...
 const availableLocales = new locale.Locales(_.pluck(languages, 'code'));
 
-module.exports = function () {
-  return function (req, res, next) {
-    let translateCookie = req.cookies.NG_TRANSLATE_LANG_KEY;
-    if (_.isArray(translateCookie)) {
-      translateCookie = translateCookie[0];
-    }
-    const localesFormReq = (translateCookie && translateCookie.replace(/"/g, '')) || req.headers['accept-language'];
+module.exports = () => (req, res, next) => {
```
Comment (Wardormeur) [inline web/lib/locale.js]: dble arrow

File: web/lib/plugins/auth.js [hunk 4]
```diff
@@ -14,22 +14,41 @@ exports.register = function (server, options, next) {
...
+              loggedInUser.user.roles.includes('cdf-admin') &&
+              cdfPath &&
+              session.target === 'cdf'
+            ) {
               return callback(null, true, { scope: 'cdf-admin' });
             }
             return callback(null, true, { scope: 'basic-user' }); // They're a `user`
           }
-          if (uErr) request.log(['error', '40x'], { status: '403', host: server.methods.getUid(), payload: request.payload, params: request.params, url: request.url, user: request.user, error: uErr }, Date.now());
+          if (uErr) {
```
Comment (Wardormeur) [inline web/lib/plugins/auth.js]: logs again

File: web/lib/http-error-handler.js (hunk 2 above)
Comment (DanielBrierton) [reply web/lib/http-error-handler.js]:
```
request.log(['error', status < 500 ? '4xx' : '5xx'], {
    status,
    host: server.methods.getUid(),
    payload: request.payload, 
    params: request.params,
    url: request.url,
    user: request.user,
    error: _.has(response, 'data.details') ? response.data.details : response.output
}, Date.now());
```

File: web/lib/plugins/auth.js [hunk 5]
```diff
@@ -14,22 +14,41 @@ exports.register = function (server, options, next) {
...
+            ? request.route.settings.auth.access[0].scope.selection.includes('cdf-admin')
+            : false;
         getUser(request, token, (uErr, loggedInUser) => {
           if (loggedInUser && !uErr) {
             // Allows to use the seneca-cdf-login token to browse normal zen,
             // but not the other way around
             request.user = loggedInUser;
-            if (loggedInUser.user.roles.indexOf('cdf-admin') > -1 &&
-               cdfPath && session.target === 'cdf') {
+            if (
```
Comment (DanielBrierton) [inline web/lib/plugins/auth.js]: Revert

File: web/lib/plugins/seneca-dojo-preloader.js [hunk 6]
```diff
@@ -6,25 +6,33 @@
...
-    preloaded.description = translater(locale, { key: '%1s in %2s',
-      var: [dojo.name, dojo.countryName] });
-    preloaded.image = [];
-    preloaded.image.push(`https://s3-eu-west-1.amazonaws.com/zen-dojo-images/${dojo.id}`);
-    preloaded['image:width'] = 300;
-    preloaded['image:height'] = 300;
-    return cb(preloaded);
-  });
+  request.seneca.act(
+    {
```
Comment (Wardormeur) [inline web/lib/plugins/seneca-dojo-preloader.js]: same issue

File: web/lib/plugins/seneca-event-preloader.js [hunk 7]
```diff
@@ -8,11 +8,12 @@ const _ = require('lodash');
 const languages = require('country-language');
 const moment = require('moment');
 
-module.exports = function (request, cb) {
+module.exports = (request, cb) => {
   const preloaded = {};
   const defaultLanguage = 'en_US';
 
-  request.seneca.act({ role: 'cd-events', cmd: 'getEvent', id: request.params.eventId },
+  request.seneca.act(
```
Comment (Wardormeur) [inline web/lib/plugins/seneca-event-preloader.js]: same issue

File: web/lib/plugins/seneca-preloader-dustjs.js [hunk 8]
```diff
@@ -1,24 +1,27 @@
...
-exports.register = function (server, options, next) {
+exports.register = (server, options, next) => {
   _.each(options.handlers, (handler) => {
     server.expose(handler, preloaders[handler]);
   });
   // Add all the server routes from the controllers.
   server.ext('onPreHandler', (request, reply) => {
-    if (_.isObject(request.route.settings.plugins.senecaPreloader) &&
-      _.isString(request.route.settings.plugins.senecaPreloader.handler)) {
+    if (
```
Comment (Wardormeur) [inline web/lib/plugins/seneca-preloader-dustjs.js]: here

File: web/lib/plugins/seneca-preloader-dustjs.js [hunk 9]
```diff
@@ -1,24 +1,27 @@
...
+      _.isObject(request.route.settings.plugins.senecaPreloader) &&
+      _.isString(request.route.settings.plugins.senecaPreloader.handler)
+    ) {
       const handler = request.route.settings.plugins.senecaPreloader.handler;
       server.plugins.senecaPreloader[handler](request, (preloaded) => {
         request.app.context.preload = preloaded;
-        request.app.context.preload.url = `${request.connection.info.protocol}://${request.info.host}${request.url.path}`;
-        request.app.context.preload.image.push('https://zen.coderdojo.com/components/cd-common/images/coderdojo-logo-light-bg.svg');
+        request.app.context.preload.url = `${request.connection.info.protocol}://${request.info
```
Comment (Wardormeur) [inline web/lib/plugins/seneca-preloader-dustjs.js]: weird stuff here

File: web/lib/locale.js [hunk 10]
```diff
@@ -1,18 +1,17 @@
...
-module.exports = function () {
-  return function (req, res, next) {
+module.exports = function requestLocale() {
+  return (req, res, next) => {
     let translateCookie = req.cookies.NG_TRANSLATE_LANG_KEY;
     if (_.isArray(translateCookie)) {
       translateCookie = translateCookie[0];
     }
-    const localesFormReq = (translateCookie && translateCookie.replace(/"/g, '')) || req.headers['accept-language'];
+    const localesFormReq =
```
Comment (Wardormeur) [inline web/lib/locale.js]: shouldn't that break on the || instead ?

File: web/lib/plugins/scooterAndBlankie.js [hunk 11]
```diff
@@ -4,26 +4,33 @@ const scooter = require('scooter');
...
-        reflectedXss: 'block',
-        scriptSrc: "'self' 'unsafe-inline' 'unsafe-eval' https://*.googleapis.com http://www.google-analytics.com https://www.google-analytics.com http://www.googletagmanager.com https://www.googletagmanager.com https://maps.gstatic.com https://www.gstatic.com https://widget.intercom.io https://js.intercomcdn.com https://www.google.com https://apis.google.com http://cdn.optimizely.com/js/3847550948.js http://www.googleadservices.com/pagead/conversion.js ",
-        styleSrc: "'self' 'unsafe-inline' http://fonts.googleapis.com https://fonts.googleapis.com",
-      } },
-    (regErr) => {
-      if (regErr) return next(regErr);
-      next();
-    });
+    server.register(
+      {
```
Comment (Wardormeur) [inline web/lib/plugins/scooterAndBlankie.js]: move brace to first row

File: web/lib/plugins/seneca-dojo-preloader.js [hunk 12]
```diff
@@ -6,21 +6,26 @@
 const translater = require('../fn/i18n-translate');
 const languages = require('country-language');
 
-module.exports = function (request, cb) {
+module.exports = (request, cb) => {
   const preloaded = {};
   const defaultLanguage = 'en_US';
-  request.seneca.act({ role: 'cd-dojos',
+  request.seneca.act({
```
Comment (Wardormeur) [inline web/lib/plugins/seneca-dojo-preloader.js]: act are pattern based : the first 2 define which message, the third is the params.
I wouldn't split the first 2, like, really.

File: web/lib/plugins/seneca-event-preloader.js [hunk 13]
```diff
@@ -8,39 +8,45 @@ const _ = require('lodash');
...
-      _.some(event.dates, (date) => {
-        const x = moment.utc(date.startTime);
-        const y = x.subtract(x.utcOffset(), 'minutes');
-        const formattedDate = new Date(date.startTime);
-        if (formattedDate > now) {
-          event.date = y.format('YYYY/MM/DD');
-          return true;
-        }
+  request.seneca.act({
+    role: 'cd-events',
```
Comment (Wardormeur) [inline web/lib/plugins/seneca-event-preloader.js]: same comment as before

File: web/lib/plugins/seneca-event-preloader.js [hunk 14]
```diff
@@ -8,39 +8,45 @@ const _ = require('lodash');
...
+    _.some(event.dates, (date) => {
+      const x = moment.utc(date.startTime);
+      const y = x.subtract(x.utcOffset(), 'minutes');
+      const formattedDate = new Date(date.startTime);
+      if (formattedDate > now) {
+        event.date = y.format('YYYY/MM/DD');
+        return true;
+      }
+    });
+    request.seneca.act({ role: 'cd-dojos', cmd: 'load', id: event.dojoId }, (dErr, dojo) => {
```
Comment (Wardormeur) [inline web/lib/plugins/seneca-event-preloader.js]: ^particularly when it's not broken down here.

File: web/lib/plugins/seneca-preloader-dustjs.js [hunk 15]
```diff
@@ -18,7 +16,9 @@ exports.register = function (server, options, next) {
       server.plugins.senecaPreloader[handler](request, (preloaded) => {
         request.app.context.preload = preloaded;
         request.app.context.preload.url = `${request.connection.info.protocol}://${request.info.host}${request.url.path}`;
-        request.app.context.preload.image.push('https://zen.coderdojo.com/components/cd-common/images/coderdojo-logo-light-bg.svg');
+        request.app.context.preload.image.push(
+          'https://zen.coderdojo.com/components/cd-common/images/coderdojo-logo-light-bg.svg',
```
Comment (Wardormeur) [inline web/lib/plugins/seneca-preloader-dustjs.js]: why is there a comma here?

**Comment by ArayB** [comment]
Closing as this is very old.

**Pull request closed without merging**